packages/genesys-spark-components/src/components/beta/gux-avatar-group/gux-avatar-groups.tsx

```diff
@@ -0,0 +1,283 @@
+import {
+  Component,
+  h,
+  JSX,
+  Element,
+  State,
+  readTask,
+  Prop,
+  writeTask
+} from '@stencil/core';
+import { afterNextRenderTimeout } from '@utils/dom/after-next-render';
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxAvatarAccent } from './../gux-avatar/gux-avatar.types';
+import { randomHTMLId } from '@utils/dom/random-html-id';
+
+/**
+ * @slot - Some gux-avatars
+ */
+@Component({
+  styleUrl: 'gux-avatar-group.scss',
+  tag: 'gux-avatar-group-beta',
+  shadow: true
+})
+export class GuxAvatarGroup {
+  @Element()
+  root: HTMLElement;
+
+  private resizeObserver?: ResizeObserver;
+  private listItemClickListeners: Map<HTMLElement, (event: Event) => void> =
+    new Map();
+
+  @Prop()
+  avatarLimit: number = 7;
```

Comment:
should this be a type `export type GuxAvatarLimit = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7;`